hbase-server/src/main/java/org/apache/hadoop/hbase/executor/EventHandler.java

```diff
@@ -96,6 +97,10 @@ public EventHandler prepare() throws Exception {
   public void run() {
     Span span = TraceUtil.getGlobalTracer().spanBuilder(getClass().getSimpleName())
       .setParent(Context.current().with(parent)).startSpan();
+    // assume that this is the top of an execution on a new or reused thread, that we're safe to
+    // blast any existing MDC state.
+    MDC.clear();
```

Comment:
We only call clear at the beginning? Is it possible to clear MDC at the end of a execution?

Reply:
Sure, seems no harm in that. You'd like to see it clear on both beginning and end?

Reply:
I prefer we do something like
```
MDC.put("blabla", "blabla");
try {
  ...
} finally {
  MDC.clear();
}
```